auction-app-frontend/src/components/tabs/Tabs.jsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import './tabs.css';
 
-const Tabs = ({ selectedTab, handleTabClick, tabs }) => {
+const Tabs = ({ selectedTab, handleTabClick, tabs, containerClassName, tabClassName, selectedTabClassName }) => {
```

Comment:
We should consider introduction of `classnameOverrides` object which can work with `classnames` package. That way we would have more flexible styling.